Tests/Editor/ScriptIssueTest.cs

```diff
@@ -10,21 +10,174 @@ namespace UnityEditor.ProjectAuditor.EditorTests
 	class ScriptIssueTest
 	{
 		private ScriptResource m_ScriptResource;
+		private ScriptResource m_ScriptResourceInPlayerCode;
+		private ScriptResource m_ScriptResourceInEditorCode;
+		private ScriptResource m_ScriptResourceIssueInNestedClass;
+		private ScriptResource m_ScriptResourceIssueInGenericClass;
+		private ScriptResource m_ScriptResourceIssueInMonoBehaviour;
+		private ScriptResource m_ScriptResourceIssueInVirtualMethod;
+		private ScriptResource m_ScriptResourceIssueInOverrideMethod;
+		private ScriptResource m_ScriptResourceIssueInCoroutine;
+		private ScriptResource m_ScriptResourceIssueInDelegate;
 		
-		[SetUp]
+		[OneTimeSetUp]
 		public void SetUp()
 		{
-			 m_ScriptResource = new ScriptResource("MyClass.cs", "using UnityEngine; class MyClass : MonoBehaviour { void Start() { Debug.Log(Camera.main.name); } }");
+			 m_ScriptResource = new ScriptResource("MyClass.cs", @"
+using UnityEngine;
+class MyClass
+{
+	void Dummy()
+	{
+		Debug.Log(Camera.main.name);
```

Comment:
I think it would be easier to understand the tests if you had a comment near by the *problematic code*.

Reply:
added comment